mkchimeenv.py

```diff
@@ -106,6 +106,42 @@ def match_opcode(opcode: int) -> Tuple[int, str, bool]:
         return (0, "Unknown", done)
 
 
+def find_requirements(path: str) -> List[Requirement]:
+    """Read and structure dependencies in a pyproject.toml file.
+
+    Parameters
+    ----------
+    path
+        Project path
+
+    Returns
+    -------
+    requirements
+        List of parsed Requirement objects
+    """
+    requirements = []
+
+    # Get the project file as a pathlib path
+    file = Path(path) / "pyproject.toml"
+
+    if not file.is_file():
+        raise FileNotFoundError(f"Project file {file} not found.")
+
+    data = toml.load(file)
+    project = data.get("project", {})
+    dependencies = project.get("dependencies", {}).copy()
+
```

Comment:
Question - should optional dependencies be included here?

Reply:
No, at least not until we need to include some of them when running `mkchimeenv`.  Everything necessary should be getting installed without them.